service_management/azure/test/integration/cloud_service/cloud_create_test.rb

```diff
@@ -15,29 +15,30 @@
 require 'integration/test_helper'
 
 describe Azure::CloudServiceManagementService do
-  include Azure::Core::Utility
-
   subject { Azure::CloudServiceManagementService.new }
+  let(:cloud_service_name) { 'testcloudservice2375' }
   let(:options) do
     {
-      location: 'West US',
-      description: 'Test'
+      location: 'East US',
+      description: 'Cloud service test description'
     }
   end
 
   before do
     Azure::Loggerx.expects(:puts).returns(nil).at_least(0)
+    VCR.insert_cassette "cloud_service/#{name}"
   end
 
-  describe '#create_cloud_service' do
-    before do
-      @cloud_name = random_string('test-service-cloud', 10)
-      subject.create_cloud_service(@cloud_name, options)
-    end
+  after do
+    subject.delete_cloud_service(cloud_service_name)
```

Comment:
Should we have delete and creation in the same place? Either both in the test or create in the "before" and delete in "after"?